backend/requirements.txt

```diff
@@ -1,3 +1,4 @@
+aiofiles==23.2.1
```

Comment:
Please consider updating the dependencies in this file patch, as some versions are outdated.